packages/documentation/src/pages/getting-started/common-tasks/routing.mdx

```diff
@@ -0,0 +1,43 @@
+# Routing
+
+## Proxy Config
+
+The proxy handles incoming URL requests to va.gov and is managed through a [proxy config](https://github.com/department-of-veterans-affairs/devops/blob/master/ansible/deployment/config/revproxy-vagov/vars/react_routes.yml) file.
+
+## React Router
+
+To use client side routing such as react-router, the `react_routes` section must be configured to ignore the route.
+For example, a React app running at `www.va.gov/appPath` would need this added:
+
+```yml
+  - path: /appPath
+    protected: no
+```
+
```

Comment:
```suggestion
By default, one url path maps to one document. This configuration changes the default behavior so that any additional path parts after `/appPath` like `/appPath/IntroductionPage` would return the document at `/appPath`.
```